main.go

```diff
@@ -184,3 +211,67 @@ func (c *Configuration) Validate() error {
 	c.VaultAdvertiseAddr = normalizeAddr(c.VaultAdvertiseAddr)
 	return nil
 }
+
+// credhubProcess iterates over the names of variables as set in the `envconfig` tag
+// on the Configuration. It prepends them with VAULT_SERVICE_BROKER_ and then looks
```

Comment:
It prepends them with `prefix`. If you want to keep the `prefix` parameter (and maybe it isn't necessary?), then I'd move the reference to VAULT_SERVICE_BROKER to the caller.

Reply:
Good catch on the comment, updated it. 

I do want to keep the prefix because in Credhub, I think it'll be easier for them to distinguish what their settings are for because of the prefix of `VAULT_SERVICE_BROKER_`. Envconfig also takes a "prefix" argument and it seems like a nice paradigm. 

I _could_ make it configurable and _default_ it to `VAULT_SERVICE_BROKER_`.